python/KAPEL.py

```diff
@@ -296,6 +297,25 @@ def record_individual_period(config, results):
         print(f"WARNING: Skipped {skipped_records} records due to missing processor count. "
                "Please set pod resource requests or specify the PROCESSORS config var.")
 
+
+def get_auth_headers(config):
+    """ Check if an authentication secret has been configured via .Values.prometheus.auth,
```

Comment:
Should be prometheus_auth:
 https://github.com/rptaylor/kuantifier/blob/master/chart/values.yaml#L81